libdebug/state/thread_context.py

```diff
@@ -91,6 +89,38 @@ def process_id(self: ThreadContext) -> int:
         """The process ID of the thread."""
         return self.context.process_id
 
+    @property
+    def pid(self: ThreadContext) -> int:
+        """The process ID of the thread."""
+        return self.context.process_id
+
+    @property
+    def thread_id(self: ThreadContext) -> int:
```

Comment:
Do we need properties for thread_id and tid?

Reply:
same as before